cnvlib/metrics.py

```diff
@@ -6,7 +6,7 @@
 from . import descriptives
 
 
-def do_metrics(cnarrs, segments=None, skip_low=False):
+def do_metrics(cnarrs, segments=None, skip_low=False): # todo: make this work in parallel?
```

Comment:
Meaning, run metrics on each cnarr in parallel?

Reply:
Yes, same as for `reference`. Both commands seem ripe for parallelization.